scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-gating-template.yaml

```diff
@@ -1,5 +1,5 @@
 - job-template:
-    name: 'cloud-mkcloud{version}-gating'
+    name: 'cloud-mkcloud{version}-{arch}-gating'
```

Comment:
Well, that's my point - I think we should have only one gating job that at some point should also run tests on non-x86. If you do a gating per architecture then you have a pass on x86 causing a submit even though aarch64 is broken.